qemu/tests/cfg/nic_opt.cfg

```diff
@@ -0,0 +1,70 @@
+- nic_opt:
+    type = nic_opt
+    only virtio_net
+    only x86_64 ppc64 ppc64le
+    # ping test setting
+    ping_package_sizes = 0 1 48 64 512 1440 1500 1505 4096 4192 32767 65507
+    # netperf test setting
+    netperf_link = netperf-2.6.0.tar.bz2
+    server_path = /var/tmp/
+    client_path = /var/tmp/
+    netperf_test_duration = 180
+    netperf_output_unit = m
+    netperf_local_cpu = yes
+    netperf_remote_cpu = yes
+    netperf_para_sessions = 1
+    test_protocols = TCP_STREAM
+    netperf_client = ${main_vm}
+    netperf_server = localhost
+    netperf_package_sizes = 1024
+    sub_type = netperf_stress
+    sub_exit_timeout = 10
+    vhost = on
+    Windows:
+        netperf_server_link_win = netserver-2.6.0.exe
+        netperf_client_link_win = netperf.exe
+        server_path_win = "c:\\"
+        client_path_win = "c:\\"
+        test_protocols = TCP_STREAM UDP_STREAM
+        netperf_package_sizes = 32 64 128 256 512 1024 2048 4096 8192 16384 32768
+    variants:
+        - indirect_desc:
+            check_type = netperf
+            variants:
+                - opt_on:
+                    nic_extra_params += ",indirect_desc=on"
+                - opt_off:
+                    nic_extra_params += ",indirect_desc=off"
+        - mrg_rxbuf:
+            check_type = netperf
+            variants:
+                - opt_on:
+                    nic_extra_params += ",mrg_rxbuf=on"
+                - opt_off:
+                    nic_extra_params += ",mrg_rxbuf=off"
+        - poll-us:
+            check_type = ping
+            variants:
+                - opt_30:
+                    netdev_extra_params_nic1 += ",poll-us=30"
```

Comment:
@sitoliu Seems this option doesn't work as expected. As the qemu command line doesn't add this option like below:

# cat 05-Host...qcow2.virtio_scsi.up.virtio_net.Guest....x86_64.io-github-autotest-qemu.nic_opt.poll-us.opt_30/debug.log

    -device virtio-net-pci,mac=9a:66:67:68:69:6a,id=idzsfQY4,vectors=4,netdev=idMET9s7,bus=pci.0,addr=0x4  \
    -netdev tap,id=idMET9s7,vhost=on,vhostfd=22,fd=9 \